### PR TITLE
Minor documentation edits

### PR DESCRIPTION
**Pull request by deanonishi**
I forgot to add my first set of changes to the '12H1' branch. I'll make sure to make future edits on the correct branch instead of master.

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -62,11 +62,11 @@ systems using ServerTemplates on the RightScale Cloud Management Platform.
 
 [app]
  RightScale application server management cookbook. This cookbook contains
- recipes that generally applicable to all application.
+ recipes that are generally applicable to all applications.
 
 [app_passenger]
- Cookbook provides apache+passenger implementation of the app LWRP. Installs and
- configures, apache+passenger application server.
+ Cookbook provides an Apache + Passenger implementation of the 'app' Lightweight Resource Provider (LWRP). Installs and
```
Comment (douglaswth) [inline README.rdoc:68]: We've been limiting the line length of the READMEs to 80 characters.

File: README.rdoc [hunk 2]
```diff
@@ -132,7 +132,7 @@ systems using ServerTemplates on the RightScale Cloud Management Platform.
 [sys_dns]
  This cookbook provides a set of dynamic DNS recipes used by RightScale
  ServerTemplates including Database Manager ServerTemplates. Cookbook currently
- supports DNSMadeEasy, DynDns, and Amazon Route53 DNS service providers.
+ supports DNSMadeEasy, DynDns, CloudDNS, and Amazon Route53 DNS service providers.
```
Comment (douglaswth) [inline README.rdoc:135]: Rackspace spells it as "Cloud DNS". Also, I think it should be DynDNS as well.

Comment (deanonishi) [inline README.rdoc:135]: Agreed. They should be changed to "DynDNS" and "Cloud DNS" respectively. Need to make sure than the same spelling is used in the input pulldown selector as well.

**Comment by douglaswth** [comment]
Hmm, this pull request was against `master`, were these changes already in a pull request against `release_12H1`?

**Pull request closed without merging**